analyzer/src/funTest/assets/projects/external/git-repo-expected-output.yml

```diff
@@ -4311,6 +4311,22 @@ analyzer:
           path: "types/node"
       curations: []
     issues:
+      NPM::isarray:2.0.5:
```

Comment:
Commit message: You mention `getPackageReferenceForMissingModule()` but you removed that function two commits earlier?

Reply:
Sorry, I had reordered comments meanwhile. I'll fix the commit message to not refer to that function anymore.